### ISSUE TITLE
Could you provide the bash file for multi-node （4machine, each 4gpus,totally 16gpus）training on IN-1K?

### ISSUE THREAD

**Issue by KimSoybean** [vturrisi/solo-learn#188]

**Comment by vturrisi** [comment]
Hey,
We never needed to run across nodes, but you can check here (https://pytorch-lightning.readthedocs.io/en/latest/clouds/cluster.html) how to do multi-node training.

**Comment by KimSoybean** [comment]
> Hey, We never needed to run across nodes, but you can check here (https://pytorch-lightning.readthedocs.io/en/latest/clouds/cluster.html) how to do multi-node training.

Thanks, I suggest that you can add the multi-nodes training to complement the solo project. VISSL and some other projects are all based on multi-nodes training.

I will try multi-node training and if finished I will PR it to solo project.

**Comment by ankitpatnala** [comment]
Sorry for commenting here.. I reckon there could be an issue with learning_rate 
https://github.com/vturrisi/solo-learn/blob/45c143dfac6e8a09258081b893726474dba04640/solo/args/utils.py#L224

Learning rate is only multiplied with batch_size and num_gpus but it should be batch_size * num_gpus * num_nodes i guess

**Comment by vturrisi** [comment]
@ankitpatnala yes, this needs to be fixed

**Comment by vturrisi** [comment]
Hey, this should be fixed in the newest version of solo. We currently support running with `horovod` and `torch.distributed.run`. If your cluster supports something different, you should check how to run there and that the learning rate is properly scaled.
